docs/Documentation/Development/Containers/apptainer.md

```diff
@@ -1,51 +1,55 @@
 ## How to use Apptainer (Singularity) on Kestrel
 
-Singularity has been deprecated in favor of a new container application called Apptainer. Apptainer is a decendent of singularity.  Apptainer will run Singularity containers and it supports Singularity commands. On Kestrel, `singularity` is an alias for `apptainer` and the two commands can be used interchangeably in most instances. However, since Singularity is deprecated, it is advised to use Apptainer.
+Singularity has been deprecated in favor of a new container runtime environment called Apptainer, which is its direct decendent. Apptainer will run Singularity containers and it supports Singularity commands by default. On Kestrel, `singularity` is an alias for `apptainer` and the two commands can be used interchangeably in most instances. However, since Singularity is deprecated, it is advised to use Apptainer.
 
-More information about Apptainer can be found at [https://apptainer.org](https://apptainer.org). 
+This page describes 
 
-Apptainer is installed on compute nodes and is accessed via a module named `apptainer`.  
+More information about Apptainer can be found at [https://apptainer.org](https://apptainer.org). 
 
-The directory `/nopt/nrel/apps/software/apptainer/1.1.9/examples`
-holds a number of containers and an example script that shows how to run containers hosting MPI programs across multiple nodes.  
+On Kestrel, Apptainer is installed on compute nodes and is accessed via a module named `apptainer` (default: `apptainer/1.1.9`). The directory `/nopt/nrel/apps/software/apptainer/1.1.9/examples` holds a number of images (`*.sif`) and an example script (`script`) that shows how to run containers hosting MPI programs across multiple nodes.  
```

Comment:
Very good point, I now explain how to check the default here instead. I err on prodding users to use specific versions of modules instead of relying on defaults for posterity, in case a version change results in unintended consequences to how the app is used.